src/plugins/gauge/components/Gauge.vue

```diff
@@ -362,7 +423,7 @@ export default {
             return this.dialLowLimitDeg >= getLimitDegree('low', 'q3');
         },
         isDialHighLimit() {
-            return this.limitHigh.length > 0 && this.dialHighLimitDeg < getLimitDegree('high', 'max');
+            return this.limitHigh.toString().length > 0 && this.dialHighLimitDeg < getLimitDegree('high', 'max');
```

Comment:
can `limitHigh` ever be null?

Reply:
Never in a normal case; this value is set by default and passed in from GaugePlugin.js, and should be handled there.

Reply:
👍